Makefile

```
@@ -221,3 +221,23 @@ change-lines:
 	sed 's/${find}/${new}/' "${filename}" > "file.tmp"
 	mv file.tmp ${filename}
 	goimports -w ${filename}
+
+.PHONY: gen-purls
+gen-purls: # Generate purls on linux os
+	@echo "==> Generating purls"
+	GOOS=linux GOARCH=amd64 go build -ldflags "$(LINKER_FLAGS)" -o $(DESTINATION)
+	go version -m $(DESTINATION) | \
+		awk '$$1 == "dep" || $$1 == "=>" { print "pkg:golang/" $$2 "@" $$3 }' | \
+		LC_ALL=C sort > compliance/purls.txt
```

You could expand this to include darwin and windows by building for each os and concatenating each resulting purls list into one
 ```suggestion
	@echo "==> Generating purls"
	GOOS=linux GOARCH=amd64 go build -ldflags "$(LINKER_FLAGS)" -o $(DESTINATION)
	go version -m $(DESTINATION) | \
		awk '$$1 == "dep" || $$1 == "=>" { print "pkg:golang/" $$2 "@" $$3 }' | \
		LC_ALL=C sort > compliance/purls-linux.txt
	# Darwin build and purl generation
	# Windows build and purl generation
	cat build/package/purls-linux.txt build/package/purls-darwin.txt build/package/purls-win.txt | LC_ALL=C sort | uniq > build/package/purls.txt
```

---

Good point, done in [522fb43](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/3374/commits/522fb43d484df6e3f33f67b5ff04a0443f0f24ee) and the purl didn't change

---

Thanks! 🚀